Persist Wishes to the localstorage #4

Eth3R-One opened the pull request:
Added feature to add new wish to local storage and get saved wishes from local storage.

atapas commented on src/AddToBucket.jsx:
```diff
@@ -3,6 +3,17 @@ import { createSignal } from "solid-js";
+        complete:false,
+      }
+    ].reverse()
+    localStorage.setItem("solid-bucket-list", JSON.stringify(allWishes))
```
@Eth3R-One Let's create a utility file for the lcalstorage related operations... u can create a util folder under src and create the utility under that.

Eth3R-One commented:
Added new utility file for local storage related operations. added two functions to get and save wishes at this moment
src/
  |__util/
            |__localStorageUtility.js

sarowarhosen03 commented:
hello sir
i have done this can i make a pR

Eth3R-One commented:
@sarowarhosen03 Yes I've made the changes and waiting for review and approval.

sarowarhosen03 commented:
> @sarowarhosen03 Yes I've made the changes and waiting for review and approval.

Oh okay thanks for letting me know

atapas commented:
@Eth3R-One Could you please resolve the merge conflicts?

Eth3R-One commented:
@atapas Dada I resolved the conflicts. Let me know if there is any issue

atapas commented on src/util/localStorageUtil.js:
```diff
@@ -0,0 +1,26 @@
+  const wishes = JSON.parse(localStorage.getItem("solid-bucket-list"));
+
+  return wishes ? wishes : dummy_Data;
+  //   if (!wishes) return dummy_Data;
```
remove the commented lines

atapas commented:
@all-contributors please add @Eth3R-One  for code

atapas commented:
@all-contributors please add @atapas for Infrastructure
